PyPDF2/pdf.py

```diff
@@ -1551,13 +1551,24 @@ def getDestinationPageNumber(self, destination):
         :rtype: int
         """
         indirectRef = destination.page
+        if type(indirectRef) is NullObject:
```

Comment:
We might want `isinstance` ([reasons](https://stackoverflow.com/a/1549854/562769)). What do you think about it?